src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int NAME_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, createReport(fromFileName));
```

Comment:
```suggestion
        String dataFromFile = readFile(fromFileName);
        String report = createReport(fromFileName);
        writeToFile(toFileName, report);
```
code like this is easier to read